.github/workflows/data_library_open_data.yml

```diff
@@ -35,9 +40,36 @@ jobs:
           AWS_ACCESS_KEY_ID: "op://Data Engineering/DO_keys/AWS_ACCESS_KEY_ID"
 
       - uses: NYCPlanning/action-library-archive@v1.1
+        if: ${{ !(inputs.use_staging_image) }}
         with:
           name: ${{ matrix.dataset }}
           latest: true
           s3: true
           compress: true
           output_format: pgdump shapefile csv
+
+      - uses: NYCPlanning/action-library-archive@v1.2
```

Comment:
Tried to do it dynamically but forgot that github does not let you specify stuff in a `uses` clause with variables. So this is a bit inelegant, but it works. Only real downside other than repeated logic is that both docker images are pulled as part of the action. But only adds about 30 s to runtime